src/server/service/import.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use grpc::{ClientStreamingSink, RequestStream, RpcContext, RpcStatus, RpcStatusCode};
+use futures::{Future, Stream};
+use futures_cpupool::{Builder, CpuPool};
+use kvproto::importpb::*;
+use kvproto::importpb_grpc;
+
+use server::metrics::*;
+use server::errors::Error;
+use raftstore::store::{UploadDir, Uploader};
+
+macro_rules! send_rpc_response {
```

Comment:
seem not necessary to define the macro here, we only use once.

Reply:
It will be used in other RPCs later.

Reply:
Like other services, we should avoid macro here.